### PR TITLE
Tutor sign up pop up #66

### LINKED ISSUE #66: [Tutor 'Sign Up' pop-up] error “Passwords do not match” is not shown when 'Confirm 'Password' field is filled with invalid data.
**Type:**
 Functional
**Environment:** Windows 10 21H2
, Chrome 110.0.5481.178
****Severity:**** Medium
**Reproducible:** Always
**Priority:** Medium

**Description**
The error message “Passwords do not match” is not shown when 'Confirm 'Password' field is filled with invalid data.

**Preconditions:**
The web-page the client is opened.
The button "Become a Tutor" is clicked and the "Tutor Sign Up" pop-up appeared.
Text fields First Name', 'Last Name', 'Email', 'Password' are filled in with valid data (e.g. Anna, Smith, anna.smith@gmail.com, qwerty1234).

**Steps to reproduce:**
Fill the text field 'Confirm 'Password' with any characters different from 'Password' (e.g. 'q').

**Actual result:**
No error is shown, system validates invalid 'Confirm 'Password' data as matching.

Image

**Expected result:**
The red error text 'Passwords do not match' is shown under 'Confirm 'Password' field when it is filled with invalid data.

### PR DESCRIPTION
**Pull request by Mygal-Alex**
what the bug looked 
image
bug fix 
image

### CODE REVIEW

**Review by AndriiNasadiuk — APPROVED** [review]
very good job

**Review by marianamartsy — APPROVED** [review]
+

**Review by LeonidShamarin — APPROVED** [review]
good job!

**Review by AndriiNasadiuk — APPROVED** [review]
good

**Review by ArtemHolikov — APPROVED** [review]
Great

File: src/containers/guest-home-page/signup-form/SignupForm.jsx [hunk 1]
```diff
@@ -92,30 +91,35 @@ const SignupForm = ({
...
 
       <AppTextField
         InputProps={passwordVisibility}
+        errorMsg={t(errors.password)}
         fullWidth
         label={t('common.labels.password')}
         onBlur={handleBlur('password')}
         onChange={handleChange('password')}
         required
-        sx={{ mb: '5px' }}
```
Comment (vlad-khrychov) [inline src/containers/guest-home-page/signup-form/SignupForm.jsx]: Is it according to mockup? I like better when margin-bottom is same for password and email. 
We can also delete `mb` for both (line 103). 
image

**Pull request merged**